soc/arm/nxp_imx/rt/soc.c

```diff
@@ -213,6 +213,13 @@ static ALWAYS_INLINE void clock_init(void)
 	CLOCK_SetMux(kCLOCK_CanMux, 2); /* Set Can clock source. */
 #endif
 
+#if defined(CONFIG_FLASH_MCUX_FLEXSPI) && !DT_PROP(DT_NODELABEL(flexspi), xip)
```

Comment:
NIT: you've removed the `xip` property so you can drop the test for it here.

Reply:
Good catch, fixed.